server.js

```diff
@@ -24,11 +26,185 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const { Schema } = mongoose;
+const songSchema = new Schema({
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+})
+
+//Reset database
+const Song = mongoose.model("Song", songSchema);
+
+if (process.env.RESET_DB) {
+  const resetDatabase = async () => {
+    await Song.deleteMany();
+    topMusicData.forEach((singleSong) => {
+      const newSong = new Song(singleSong);
+      newSong.save();
+    });
+  }
+  resetDatabase();
+  // call a function while declaring it - extra curriculum 
+}
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+
+  res.send({
+    Message: "Browse in top music data titles, below you find the different endpoints",
+    Routes: [{
+      "/songs": "This engpoint returns a list of top music titles",
+      "/songs/id/:id" : "This engpoint returns a specific music item by music id. exampel: /songs/id/2",
+      "/songs/artist/:artistname" :"This endpoint returns a list of the music titles by a specific artist name, exampel: Lady Gaga",
+      "/songs/genre&danceability": "This endpoint returns a list of the top music titles in a specific genre, exampel: /songs?genre=pop&danceability=76",
+      
+    }]
+  });
+
+});
+
+
+//Route1 : all songs from database
+app.get("/songs", (request, response) => {
+  const songes = topMusicData;
+  if (songes) {
+    response.status(200).json({
+      success: true,
+      message: "OK",
+      body: {
+        topMusicData: songes
+      }
+    })
+  } else {
+    response.status(500).json({
```

Comment:
In the /songs route, if the songes variable is empty, it returns a 500 status code instead of 404. It would be more appropriate to return a 404 status code since the songs are not found.